packages/data-stores/src/help-center/actions.ts

```diff
@@ -39,6 +33,17 @@ export const setIsMinimized = ( minimized: boolean ) =>
 		minimized,
 	} as const );
 
+export const setShowHelpCenter = function* ( show: boolean ) {
+	if ( ! show ) {
+		yield setIsMinimized( false );
+	}
```

Comment:
Just needs a code comment.

`// reset minimized state when the help center is closed`